### DISCUSSION TITLE
iPhone OS 14.5 PWA + getUserMedia video lagging

### DISCUSSION THREAD

**Discussion by ChouchouCendre** [vimeo/player.js#1009] [Bug reports]
### Expected Behavior
Video playing without lagging

### Actual Behavior
Video lagging, very slow

### Steps to Reproduce
- On an iPhone with OS 14.5, go to [https://codepen.io/sunlimet/pen/yLbgOEx](https://codepen.io/sunlimet/pen/yLbgOEx)
- Button share, then "Add to Home Screen"
- Open PWA, press play = video is lagging

As you can see, there's only the Vimeo iframe and a navigator.mediaDevices.getUserMedia

- Android do not have this issue
- Can reproduce on old IPhones (SE first and second generation, iPhone 6), but it's ok on iPhone 11
- Tested with YouTube, it's ok
- In Safari standalone (not PWA), it is better but it's also lagging

**Comment by lssamlyon** [comment]
Thanks for raising this with us, I have made of note of this ticket internally and will update here once I have some more details.
